src/vs/workbench/api/browser/mainThreadNotebookDocuments.ts

```diff
@@ -213,7 +213,6 @@ export class MainThreadNotebookDocuments implements MainThreadNotebookDocumentsS
 					this._instantiationService,
 					uri,
 					undefined,
-					options.viewType
```

Comment:
I hardcoded `viewType` as `'jupyter-notebook'` for Positron notebooks. From my understanding, we don't want to support multiple view types (like GitHub issue notebooks) in our UI, and it keeps the code simpler. This is also how the other notebook UIs in Code OSS do things (interactive window and the new repl edito).